hack/preview-template.env

```diff
@@ -30,19 +30,8 @@ export JVM_BUILD_SERVICE_IMAGE_REPO=
 export JVM_BUILD_SERVICE_IMAGE_TAG=
 export JVM_BUILD_SERVICE_PR_OWNER=
 export JVM_BUILD_SERVICE_PR_SHA=
-export JVM_BUILD_SERVICE_CACHE_IMAGE_REPO=
-export JVM_BUILD_SERVICE_CACHE_IMAGE_TAG=
-export JVM_BUILD_SERVICE_SIDECAR_IMAGE_REPO=
-export JVM_BUILD_SERVICE_SIDECAR_IMAGE_TAG=
-export JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_REPO=
-export JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_TAG=
-export JVM_DELETE_TASKRUN_PODS=
-export JVM_BUILD_SERVICE_JDK8_BUILDER_IMAGE_REPO=
-export JVM_BUILD_SERVICE_JDK8_BUILDER_IMAGE_TAG=
-export JVM_BUILD_SERVICE_JDK11_BUILDER_IMAGE_REPO=
-export JVM_BUILD_SERVICE_JDK11_BUILDER_IMAGE_TAG=
-export JVM_BUILD_SERVICE_JDK17_BUILDER_IMAGE_REPO=
-export JVM_BUILD_SERVICE_JDK17_BUILDER_IMAGE_TAG=
```

Comment:
I checked jvm-build-service and didn't find any reference to these env vars, so I assumed they are not used anymore. cc @stuartwdouglas for double check please